README.md

```diff
@@ -117,6 +117,31 @@ If you use `-skip-dnssec`, letsdane will use the Authenticated Data flag.
 
 I wanted to try DANE, but no browser currently supports it. It may still be a long way to go for browser support, but if you want to try it now you can!
 
+
+## Docker Usage
+
+### Docker CLI
+
+```bash
+docker run -dp 8080:8080 \
+  -v "$(pwd)"/dane:/root/.letsdane \
+  --restart unless-stopped \
+  anunayj/letsdane -verbose
+```
+### Docker-compose

```

Comment:
It's easy to figure out how to use docker-compose based on the docker run example so there is no need for this section. it's better to keep the readme short